libs/react-helpers/src/lib/format/number.ts

```diff
@@ -19,11 +19,11 @@ export function toBigNum(
 
 export function addDecimal(
   value: string | number | EthersBigNumber,
-  decimals: number,
+  decimals?: number,
```

Comment:
question (blocking): I'm uncomfortable with this change. This function is used a lot to add and remove decimals specifically for price and sizes. Its important that decimals is ALWAYS provided so that this can be done accurately. No someone could easily omit this which could result in transactions being sent with incorrect values.

Reply:
updated.